pkg/rc/farm.go

```diff
@@ -13,10 +13,12 @@ import (
 	"github.com/square/p2/pkg/kp/rcstore"
 	"github.com/square/p2/pkg/labels"
 	"github.com/square/p2/pkg/logging"
+	p2metrics "github.com/square/p2/pkg/metrics"
```

Comment:
How strongly do you feel about the name collision introduced by this package? I note that `p2/pkg/metrics` and `rcrowley/go-metrics` are almost always guaranteed to appear together. If we were to change the name of our package, now is the time to do it.

Of course, I can't think of a better name!